tests/robustness/traffic/key_window.go

```diff
@@ -0,0 +1,62 @@
+package traffic
+
+import (
+	"fmt"
+	"math/rand"
+	"sync"
+)
+
+type KeyWindow struct {
+	sync.Mutex
+
+	Counter    int
+	WindowSize int
+	Keys       []int
+	keyPrefix  string
+}
+
+func NewKeyWindow(size int, keyPrefix string) *KeyWindow {
+	k := &KeyWindow{
+		Counter:    0,
+		WindowSize: size,
```

Comment:
can we instead use len(Keys) when we need the size?

Reply:
+1